lib/core/exceptions.toit

```diff
@@ -150,10 +150,10 @@ class Exception_:
 
 // Spontaneous entry points invoked by the interpreter when failure occurs at runtime.
 // We use 'rethrow' to avoid extra frame in trace.
-lookup_failure_ receiver selector_or_selector_offset:
+lookup_failure_ receiver selector_offset:
```

Comment:
When we have a method call for a selector that doesn't exist in the whole system, then we call `lookup_failure_` directly (a static call). In that case we pass the name of the selector as a string.

Reply:
Reverted my change.